src/index.html

```diff
@@ -7,6 +7,41 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="container">
```

Comment:
for what you crated this container?

Reply:
it's easier that way

Reply:
> 

Its better to add mix to our calendar rather then create new blocks